console/console.go

```diff
@@ -0,0 +1,103 @@
+package console
+
+import (
+	consolev1alpha1 "github.com/openshift/api/console/v1alpha1"
```

Comment:
can we separate imports like https://github.com/red-hat-storage/odf-operator/blob/main/controllers/subscriptions.go#L19